### ISSUE TITLE
FAIMS splitting for PTM localisation

### ISSUE THREAD

**Issue by SC0O8Y** [Nesvilab/FragPipe#663]
Hi Fengchao, 

I hope you are well. 

I searched through github request to see if this is a duplicate, I couldn't find it so I hope it hasn't been asked. 

I am trying to do localisation analysis of uncharacterised **PTM X**. 

My files were acquired on a Exploris 480 and two CVs were used. I used the FAIMS splitter and realised that localisation doesn't work with the mzXML files. As such I was able to make fragpipe run and instead enabled PTM-Sheppard and I am attempting a run, do you think this will work for localising the mass shifts? 

Kindest regards  (Sorry for the lack of formality in my previous posts)
Joel

**Comment by kabalak** [comment]
Another alternative could be a simple CV-specific export of the raw file from the Thermo FreeStyle tool (cf. https://github.com/Nesvilab/FragPipe/issues/662#issuecomment-1110588583) - so you would work on the .raw level and would not need a double conversion.

**Comment by SC0O8Y** [comment]
Hmm, so manually split all run files in freestyle? May be a bit tedious
when running a large batch of samples.

**Comment by fcyu** [comment]
I agree. Feel free to let know if someone knows how to do it programmatically.

Best,

Fengchao

**Comment by SC0O8Y** [comment]
No worries mate, I'll ask around.

:-)

**Comment by chhh** [comment]
@SC0O8Y @fcyu PNNL has a tool to split raw files by CV: https://github.com/PNNL-Comp-Mass-Spec/Thermo-FAIMS-to-MzML
